timed/employment/tests/test_location.py

```diff
@@ -1,19 +1,25 @@
+import pytest
 from django.urls import reverse
 from rest_framework import status
 
 from timed.employment.factories import LocationFactory
+from timed.projects.factories import CustomerAssigneeFactory
 
 
-def test_location_list(auth_client):
+@pytest.mark.parametrize("is_customer, expected", [(True, 0), (False, 1)])
+def test_location_list(auth_client, is_customer, expected):
+    if is_customer:
+        CustomerAssigneeFactory.create(user=auth_client.user, is_customer=True)
```

Comment:
Same here - test also with a `CustomerAssignee` on the current user that has `is_customer=False`